app/helpers/application_helper.rb

```diff
@@ -32,6 +32,13 @@ def link_to(*args, &block)
     end
   end
 
+  def link_to_function(name, function, html_options = {})
+    onclick = "#{"#{html_options[:onclick]}; " if html_options[:onclick]}#{function}; return false;"
```

Comment:
Could you split out some of the interpolation here?  It's incredibly difficult to read.